src/index.tsx

```diff
@@ -212,7 +212,11 @@ export class RCPhoneInput extends React.Component<IProps, IState> {
     if (props.value) {
```

Comment:
use `value` here as you already destructed it

Reply:
`if (props.value) {` is not on the same function